DEVELOP.md

```diff
@@ -10,7 +10,7 @@ Run `yarn install` to install the npm dependencies.
 
 Our [`package.json`](package.json) houses a collection of [run-scripts](https://docs.npmjs.com/cli/run-script) that we use to maintain, test, build, and publish the OVH Manager, notably:
 
-* `split` runs `scripts/split.js`, which generate a subtree branch (from the current one) for the specified package.
+-   `split` runs `scripts/split.js`, which generate a subtree branch (from the current one) for the specified package.
```

Comment:
Theses changes aren't necessary since we turned off this rule (ref: https://github.com/ovh-ux/manager/blob/master/.remarkrc#L4)

Reply:
i added it in ignore file again